src/Flash.js

```diff
@@ -1,31 +1,35 @@
-import React from 'react'
 import PropTypes from 'prop-types'
-import classnames from 'classnames'
-import sass from 'sass.macro'
-import {injectGlobal} from 'emotion'
 import styled from 'styled-components'
-import {COMMON} from './constants'
+import {COMMON, get} from './constants'
 import theme from './theme'
 
-injectGlobal(sass`
-  @import "primer-alerts/index.scss";
-`)
-
 const schemeMap = {
-  green: 'success',
-  red: 'error',
-  yellow: 'warn'
+  green: {color: 'colors.green.8', bg: 'colors.green.1', borderColor: 'colors.blackfade15'},
```

Comment:
Might be cleaner to just use class names here like I did with https://github.com/primer/components/pull/380/files#diff-89d2c1586a15b25e428f4cbfe45590f6R44 🤔

Reply:
cc @mxstbr what do you feel like is a better practice/cleaner?